circuitpython_library_pypi_stats.md

```diff
@@ -3,6 +3,10 @@
 Here is a listing of current Adafruit CircuitPython libraries download statistics.  
 There are 308 libraries available.
 
+
+Total Blinka downloads: 34218  
+Total PyPI library downloads: 182737
+
 | PyPI Package | Downloads in the Last 7 Days |
 | --- | --- |
 | adafruit-circuitpython-busdevice | 28409 downloads |
```

Comment:
This would look like the following.
```
| Adafruit CircuitPython BusDevice (adafruit-circuitpython-busdevice) | 28409 downloads |
```
It's longer, and a little redundant, but I think it reads better. Let me know what you think or if you have any questions.